auto_training/export.py

```diff
@@ -129,11 +129,11 @@ def parse_args():
 
     if args.shape is None:
         img_scale = cfg.test_pipeline[1]['img_scale']
-        input_shape = (1, 3, img_scale[1], img_scale[0])
+        input_shape = (4, 3, img_scale[1], img_scale[0])
     elif len(args.shape) == 1:
-        input_shape = (1, 3, args.shape[0], args.shape[0])
+        input_shape = (4, 3, args.shape[0], args.shape[0])
     elif len(args.shape) == 2:
-        input_shape = (1, 3) + tuple(args.shape)
+        input_shape = (4, 3) + tuple(args.shape)
```

Comment:
are these changes needed if we use dynamic export anyway?

Reply:
thanks, we don`t use this param at dynamic mode.